diff --git a/node_modules/hey-listen/dist/hey-listen.es.js b/node_modules/hey-listen/dist/hey-listen.es.js
index 320b49c..d66313f 100644
--- a/node_modules/hey-listen/dist/hey-listen.es.js
+++ b/node_modules/hey-listen/dist/hey-listen.es.js
@@ -1,16 +1,16 @@
-var warning = function () { };
-var invariant = function () { };
-if (process.env.NODE_ENV !== 'production') {
-    warning = function (check, message) {
-        if (!check && typeof console !== 'undefined') {
-            console.warn(message);
-        }
-    };
-    invariant = function (check, message) {
-        if (!check) {
-            throw new Error(message);
-        }
-    };
+var warning = function () {};
+var invariant = function () {};
+if (typeof process !== 'undefined' && process.env.NODE_ENV !== 'production') {
+  warning = function (check, message) {
+    if (!check && typeof console !== 'undefined') {
+      console.warn(message);
+    }
+  };
+  invariant = function (check, message) {
+    if (!check) {
+      throw new Error(message);
+    }
+  };
 }
 
 export { invariant, warning };
diff --git a/node_modules/hey-listen/dist/hey-listen.js b/node_modules/hey-listen/dist/hey-listen.js
index 2e297aa..3e1af51 100644
--- a/node_modules/hey-listen/dist/hey-listen.js
+++ b/node_modules/hey-listen/dist/hey-listen.js
@@ -1,24 +1,26 @@
 (function (global, factory) {
-  typeof exports === 'object' && typeof module !== 'undefined' ? factory(exports) :
-  typeof define === 'function' && define.amd ? define(['exports'], factory) :
-  (global = global || self, factory(global.heyListen = {}));
-}(this, function (exports) { 'use strict';
+  typeof exports === 'object' && typeof module !== 'undefined'
+    ? factory(exports)
+    : typeof define === 'function' && define.amd
+    ? define(['exports'], factory)
+    : ((global = global || self), factory((global.heyListen = {})));
+})(this, function (exports) {
+  'use strict';
 
-  exports.warning = function () { };
-  exports.invariant = function () { };
-  if (process.env.NODE_ENV !== 'production') {
-      exports.warning = function (check, message) {
-          if (!check && typeof console !== 'undefined') {
-              console.warn(message);
-          }
-      };
-      exports.invariant = function (check, message) {
-          if (!check) {
-              throw new Error(message);
-          }
-      };
+  exports.warning = function () {};
+  exports.invariant = function () {};
+  if (typeof process !== 'undefined' && process.env.NODE_ENV !== 'production') {
+    exports.warning = function (check, message) {
+      if (!check && typeof console !== 'undefined') {
+        console.warn(message);
+      }
+    };
+    exports.invariant = function (check, message) {
+      if (!check) {
+        throw new Error(message);
+      }
+    };
   }
 
   Object.defineProperty(exports, '__esModule', { value: true });
-
-}));
+});
diff --git a/node_modules/hey-listen/dist/hey-listen.min.js b/node_modules/hey-listen/dist/hey-listen.min.js
index 588c456..f734730 100644
--- a/node_modules/hey-listen/dist/hey-listen.min.js
+++ b/node_modules/hey-listen/dist/hey-listen.min.js
@@ -1 +1,20 @@
-!function(n,e){"object"==typeof exports&&"undefined"!=typeof module?e(exports):"function"==typeof define&&define.amd?define(["exports"],e):e((n=n||self).heyListen={})}(this,function(n){"use strict";n.warning=function(){},n.invariant=function(){},"production"!==process.env.NODE_ENV&&(n.warning=function(n,e){n||"undefined"==typeof console||console.warn(e)},n.invariant=function(n,e){if(!n)throw new Error(e)}),Object.defineProperty(n,"__esModule",{value:!0})});
+!(function (n, e) {
+  'object' == typeof exports && 'undefined' != typeof module
+    ? e(exports)
+    : 'function' == typeof define && define.amd
+    ? define(['exports'], e)
+    : e(((n = n || self).heyListen = {}));
+})(this, function (n) {
+  'use strict';
+  (n.warning = function () {}),
+    (n.invariant = function () {}),
+    typeof process !== 'undefined' &&
+      'production' !== process.env.NODE_ENV &&
+      ((n.warning = function (n, e) {
+        n || 'undefined' == typeof console || console.warn(e);
+      }),
+      (n.invariant = function (n, e) {
+        if (!n) throw new Error(e);
+      })),
+    Object.defineProperty(n, '__esModule', { value: !0 });
+});
diff --git a/node_modules/hey-listen/dist/index.js b/node_modules/hey-listen/dist/index.js
index d3e2bc7..b768257 100644
--- a/node_modules/hey-listen/dist/index.js
+++ b/node_modules/hey-listen/dist/index.js
@@ -2,17 +2,17 @@
 
 Object.defineProperty(exports, '__esModule', { value: true });
 
-exports.warning = function () { };
-exports.invariant = function () { };
-if (process.env.NODE_ENV !== 'production') {
-    exports.warning = function (check, message) {
-        if (!check && typeof console !== 'undefined') {
-            console.warn(message);
-        }
-    };
-    exports.invariant = function (check, message) {
-        if (!check) {
-            throw new Error(message);
-        }
-    };
+exports.warning = function () {};
+exports.invariant = function () {};
+if (typeof process !== 'undefined' && process.env.NODE_ENV !== 'production') {
+  exports.warning = function (check, message) {
+    if (!check && typeof console !== 'undefined') {
+      console.warn(message);
+    }
+  };
+  exports.invariant = function (check, message) {
+    if (!check) {
+      throw new Error(message);
+    }
+  };
 }
